widgetastic_manageiq/__init__.py

```diff
@@ -5995,3 +5995,8 @@ def __init__(self, parent, locator, logger=None):
 
     def read(self):
         return [self.browser.text(el) for el in self.browser.elements(self.ITEMS)]
+
+
+class ImportExportFlashMessages(FlashMessages):
+    ROOT = ".//div[contains(@class, 'import-flash-message')]"
+    MSG_LOCATOR = './/div[contains(@class, "alert")]'
```

Comment:
Use the same string format for both, e.g.,

```
    ROOT = ".//div[contains(@class, 'import-flash-message')]"
    MSG_LOCATOR = ".//div[contains(@class, 'alert')]"
```

Reply:
> Marking WIPTEST while @tpapaioa's comments are addressed

Done

Reply:
`FileImportSelectorView` and `AutomateImportExportBaseView` haven't been updated to use `ImportExportFlashMessages`.